### PR TITLE
Set up the CI for format and build checks

### LINKED ISSUE #3: Add a linter/compiler on the CI
A linter/compiler like https://github.com/uber/prototool could be added as a GitHub Actions to check that the files compile and are well formatted.
Currently our formatting is poor, for example the indentation is not consistent.

### PR DESCRIPTION
**Pull request by hug-dev**
Format the protobuf files and modifies the README.md file.

Fixes #3

### CODE REVIEW

File: ci/check_format.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+OUT_DIR="/tmp/parsec-operations-ci"
+mkdir -p $OUT_DIR
+
+# protobuf tests
+mkdir -p $OUT_DIR/protobuf
+PROTOBUF_FILES=protobuf/*.proto
+for f in $PROTOBUF_FILES
+do
+	clang-format $f > $OUT_DIR/$f
+	git diff $f $OUT_DIR/$f
```
Comment (ionut-arm) [inline ci/check_format.sh]: Why `git diff` and not `diff`? Does the same, return code 1 if they differ

Comment (hug-dev) [reply ci/check_format.sh]: Because the output of `git diff` is prettier and is the one commonly used (even in Markdown).

Comment (ionut-arm) [reply ci/check_format.sh]: Fair enough!

**Comment by hug-dev** [comment]
Let's see if the CI passes!

**Pull request merged**